application/org.openjdk.jmc.console.agent/plugin.xml

```diff
@@ -0,0 +1,99 @@
+   provided that the following conditions are met:
+   
+   1. Redistributions of source code must retain the above copyright notice, this list of conditions
+   and the following disclaimer.
+   
+   2. Redistributions in binary form must reproduce the above copyright notice, this list of
+   conditions and the following disclaimer in the documentation and/or other materials provided with
+   the distribution.
+   
+   3. Neither the name of the copyright holder nor the names of its contributors may be used to
+   endorse or promote products derived from this software without specific prior written permission.
+   
+   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+   IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+   FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+   CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+   DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+   DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+   WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+   WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+<?eclipse version="3.4"?>
+<plugin>
+    <extension point="org.openjdk.jmc.rjmx.actionProvider">
+        <action
+            factory="org.openjdk.jmc.console.agent.actions.AgentEditorOpener"
+            icon="icons/agent-16.png"
+            label="Agent Plugin">
+        </action>
+        <provider
+            icon="icons/agent-16.png"
+            id="org.openjdk.jmc.console.ext.agent"
+            description="The JMC Agent Plugin can be used to add JFR instrumentation declaratively to a running program"
+            label="Agent Plugin"
+            priority="101"
+            doubleClickActionIndex="0">
+            <action
+                factory="org.openjdk.jmc.console.agent.actions.AgentEditorOpener"
+                icon="icons/agent-16.png"
+                label="Open the JMC Agent Plugin">
+            </action>
+        </provider>
+    </extension>
+
+    <extension point="org.eclipse.ui.editors">
+        <editor
+            class="org.openjdk.jmc.console.ext.agent.editor.AgentEditor"
+            default="true"
+            id="org.openjdk.jmc.console.ext.agent.editor.AgentEditor"
+            icon="icons/agent-16.png"
+            name="Agent Plugin"/>
+    </extension>
+    
+    <extension point="org.eclipse.ui.editors">
+      	<editor
+            id="org.openjdk.jmc.console.ext.agent.tabs.raweditor.RawEditor"
+            name="Agent XML Editor"
+            icon="icons/agent-16.png"
+            extensions="xml"
+            class="org.openjdk.jmc.console.ext.agent.tabs.raweditor.RawEditor"
```

Comment:
Same here, shouldn't it be `org.openjdk.jmc.console.agent.raweditor.RawEditor` ?